launchUI.py

```diff
@@ -2143,10 +2143,15 @@ def create_layout_pannel(master, x, width, height):
 
 def clickDisplayLayout():
     # Hardcodes the related path, will fix later.
-    flow_basePath = os.path.dirname(os.path.abspath(__file__)) + "/tools/OpenROAD-flow-scripts/flow/"
-    layout_path = flow_basePath + "layout.png"
-    odb_path = flow_basePath + "results/nangate45/gcd/base/6_final.odb"
-    cmd_path = flow_basePath + "cmd.tcl"
+    test_module_name = "TileRTL__f6add65c4dde1319"

```

Comment:
You are talking about make this by default `CGRARTL`, right?
PyMTL by default generates RTL file with a random ending. Can you just locate the file with prefix of `CGRARTL__`. I think that would work?

Reply:
Yeah, I have updated it

Reply:
Did you make the change/update in this file? I didn't see the code changed.

Reply:
Have uploaded now.